roles/splunk/defaults/main.yml

```diff
@@ -13,11 +13,19 @@ splunk_package_url_uf: "https://download.splunk.com/products/universalforwarder/
 splunk_download_local: true  # This defines how the download process works. If `true` it will download to localhost and copy around to hosts from there. If `false` each host will download the package individually.
 splunk_install_type: undefined # There are two ways to configure this. The easiest way is to nest hosts under either a "full" group or a "uf" group in your inventory and main.yml will handle it for you. Or, you can also set the value via a group_vars or host_vars file.
 splunk_install_path: /opt # Base directory on the operating system to which splunk should be installed
+splunk_file_type: tgz # Default is a tgz installer
+splunk_package_gpg: SplunkPGPKey.pub
 least_privileged: false # Do not change. This get automatically set in `tasks/main.yml` based on the version and install type.
-splunk_nix_user: splunk
-splunk_nix_group: splunk
 local_os_user: false # Whenther or not to force creation of a user using the `luseradd` or not.
 local_os_group: false # Whether or not to force creation of a group using the `lgroupadd` or not.
+splunk_nix_user: splunk
+splunk_nix_group: splunk
+splunk_server_sslVersions: tls1.2
+splunk_web_sslVersions: tls1.2
+splunk_webserver_port: 8000
+splunk_licenses: []
```

Comment:
there is a different variable for this [here](https://github.com/splunk/ansible-role-for-splunk/blob/master/roles/splunk/defaults/main.yml#L22)